dpx/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![feature(ptr_wrapping_offset_from)]
 #![feature(c_variadic)]
 #![feature(const_transmute)]
+#![feature(new_uninit)]
```

Comment:
Is this being used somewhere? I don't see `new_uninit` or `assume_init` in the diff.

Reply:
`read_APP1_Exif` in dpx/src/dpx_jpegimage.rs, d3cc25c.

Reply:
https://github.com/crlf0710/tectonic/commit/d3cc25c550cc9cfe88eac69a1b65252465c18229#r35633404